CONFIGURATION.md

```diff
@@ -0,0 +1,72 @@
+Different environment variable types are explained at [django-environ](https://github.com/joke2k/django-environ#supported-types).
+
+## Common
+
+* `SECRET_KEY`: A secret key used for cryptography. This needs to be a random string of a certain length. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-SECRET_KEY).
+* `ALLOWED_HOSTS`: A list of hosts/domains your service will be served from. See [more](https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts).
+
+## Database
+
+Per default [Sqlite3](https://sqlite.org/) is used as database for simple deployment and stored at `/var/lib/document-merge-service/data/sqlite3.db`. Create a volume to make it persistent.
+
+To scale the service a different database storage is needed. Any database supported by [Django](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-DATABASE-ENGINE) can be used.
+
+* `DATABASE_ENGINE`: Database backend to use.
+* `DATABASE_HOST`: Host to use when connecting to database
+* `DATABASE_PORT`: Port to use when connecting to database
+* `DATABASE_NAME`: Name of database to use
+* `DATABASE_USER`: Username to use when connecting to the database
+* `DATABASE_PASSWORD`: Password to use when connecting to database
+
+## Unoconv
+
+* `UNOCONV_ALLOWED_TYPES`: list of types allowed to convert to. See `unoconv --show` (default: ['pdf'])
+* `UNOCONV_SERVER`: hostname or ip of unoconv listener. If no server is specified a local unoconv
+   listener inside the container is used. Defaults to `None`.
+* `UNOCONV_PORT`: port of unoconv listener. Only relevant if `UNOCONV_SERVER` is set. Defaults to `2002`.
+* `UNOCONV_PYTHON`: string defaults to "/usr/bin/python3.5"
+* `UNOCONV_PATH`: string defaults to "/usr/bin/unoconv"
+
+## python-docx-template
+* `DOCXTEMPLATE_JINJA_EXTENSIONS`: list of [jinja2 extensions](http://jinja.pocoo.org/docs/2.10/extensions/) to load
+
+In python-docx-template following additional custom filters are implemented:
+
+* multiline(value) - wraps the value in a [Listing](https://docxtpl.readthedocs.io/en/latest/#escaping-newline-new-paragraph-listing) for multiline support
+* datetimeformat(value, format, locale)
+* dateformat(value, format, locale)
+* timeformat(value, format, locale)
+* getwithdefault(value, default) - converts None to empty string (or provided default value) or leaves strings as is
+* emptystring(value) - converts None to empty string or leaves strings as is (deprecated in favor of getwithdefault)
+
+For formatting use babel and its uniode compatible [format](http://babel.pocoo.org/en/latest/dates.html#date-fields).
+
+## Authentication / Authorization
+
+By default no authentication is needed. To protect api, integrate it with your [IAM](https://en.wikipedia.org/wiki/Identity_management) supporting Open ID Connect. If not availbale you might consider using [Keycloak](https://www.keycloak.org/).
```

Comment:
I believe there is a `the` missing in front of `api`. 
Also `s/availbale/available`.